core/src/main/scala/cats/data/Streaming.scala

```diff
@@ -887,7 +887,7 @@ trait StreamingInstances extends StreamingInstances1 {
       def combine[A](xs: Streaming[A], ys: Streaming[A]): Streaming[A] =
         xs concat ys
 
-      override def map2[A, B, Z](fa: Streaming[A], fb: Streaming[B])(f: (A, B) => Z): Streaming[Z] =
+      override def map2[A, B, Z](fa: Streaming[A], fb: Streaming[B])(f: (A, B) => Z)(implicit ev: Monoidal[Streaming]): Streaming[Z] =
```

Comment:
Why was this change necessary? If you have a `Streaming` in hand you can easily do `map` or `flatMap` without needing a `Monoidal` instance.